### PR TITLE
New project dialog

### LINKED ISSUE #8243: Add docs for OrchardCore VS Templates

### PR DESCRIPTION
**Pull request by agriffard**
Add a screenshot of the 'New project' dialog in Visual Studio in the project templates documentation page.

Fixes #8243

new-project

### CODE REVIEW

File: src/docs/getting-started/templates/README.md [hunk 1]
```diff
@@ -72,7 +72,9 @@ dotnet new ocmvc
 
 ### From Visual Studio (New Project dialog)
 
-The templates can also be used from the New Project dialog in Visual Studio.
+The templates can also be used from the 'New Project' dialog in Visual Studio.
```
Comment (hishamco) [inline src/docs/getting-started/templates/README.md:75]: It would be nice if we specify the VS version, coz this is a new feature AFAIK

**Review by hishamco — CHANGES_REQUESTED** [review]
LGTM after specify the VS version

**Comment by agriffard** [comment]
new-project-option

**Review by hishamco — APPROVED** [review]
Now look much better

**Pull request merged**